plugins/synapticsmst/synapticsmst.quirk

```diff
@@ -1,3 +1,27 @@
+[SynapticsMSTBoardID=0]
+Name = Synaptics EVB development board
+GuidTemplate = evb
+
+[SynapticsMSTBoardID=272]
```

Comment:
Ya, I'd much rather see `SynapticsMSTBoardID=0x0272` -- `=272` seems a little magic

Reply:
You mean = 0x110, but yeah understood.

Reply:
If files in the wild are %u then ignore me as above.

Reply:
Well I think if it makes it more readable in the quirk file keeping it as hex there would be fine, it just would be converted to decimal when the GUID is built.

Right now I figured consistency is better.
So your call if you want me to
* keep it as is (decimal everywhere)
* switch to hex in the quirk file and convert to decimal only when GUID is built

Reply:
> Right now I figured consistency is better

Agreed.